### ISSUE TITLE
File Still Exists in Front End After Deleted Locally

### ISSUE THREAD

**Issue by jmwright** [stemn/stemn-frontend#9]
[This file](https://stemn.com/files/5b15410c252c9400192ab940/5b19e761b2698100194c802d?revision=5b19f01cb2698100194c809c) in my project has been deleted locally, but is still present in the front end. I tried doing a commit, but that didn't fix it. Am I missing a step?

**Comment by MrBlenny** [comment]
Nope, if it's deleted from your dropbox it should get deleted here also. 
Must be a dropbox syncing error. @tomnlittle or @sabrehagen will take a look.

**Comment by jmwright** [comment]
Thanks

Looks like I've got another file that isn't syncing properly as well: https://stemn.com/files/5b15410c252c9400192ab940/5b19f2c1b2698100194c80cf?revision=5b1af08fb2698100194c8a13

**Comment by sabrehagen** [comment]
Not missing a step, no. The file should be deleted from Stemn's file listing. Investigating...
